app/helpers/application_helper/title.rb

```diff
@@ -85,6 +85,8 @@ def title_from_layout(layout)
         _("Utilization")
       when /^miq_request/
         _("Requests")
+      when "configuration_script"
```

Comment:
it's likely because the layout name isn't configuration_script.  Can you print out what the layout name is here?

Reply:
Managed to fix the title in this pr: https://github.com/ManageIQ/manageiq/pull/23032.

This file change is still needed however as this handles the title in the browser tab name.